code/modules/mob/living/carbon/human/species.dm

```diff
@@ -599,6 +599,10 @@
 /datum/species/proc/spec_death(gibbed, mob/living/carbon/human/H)
 	return
 
+//called when acquiring a mutation, return TRUE to prevent the mutation
+/datum/species/proc/spec_mutation(mob/living/carbon/human/H)
```

Comment:
i thought it could be done when a race actually uses it, but i'll add it if you want